source/includes/code-examples/faq.cs

```diff
@@ -13,4 +13,21 @@ async Task CreateIndexesAsync(IMongoDatabase database)
     await database.GetCollection<Movie>("movies")
           .Indexes.CreateOneAsync(moviesIndex);
 }
-// end-create-index
\ No newline at end of file
+// end-create-index
+
+// start-atlas-search
+// Client used to set up your DbContext 
+var client = new MongoClient("<connection string>");
+
+var clientDB = client.GetDatabase("sample_guides");
+var collection = clientDB.GetCollection<Planet>("planets");
+
+var searchResult = collection.Aggregate()
+    .Search(Builders<Planet>.Search.Equals(p => p.hasRings, true))
```

Comment:
Q: (nit) this is not a very illustrative atlas search example, as it could be easily performed with a regular query. Consider changing this example?

Reply:
I wanted to keep the method itself pretty simple so that the focus is on the fact that you can call the driver methods (.Aggregate(), .Search()) rather than on the implementation of search itself since that's documented on the linked C# page.